compiler/quilt/tools/command.py

```diff
@@ -1090,48 +1121,53 @@ def access_add(package, user):
     """
     Add access
     """
-    owner, pkg = parse_package(package)
-    session = _get_session()
+    team, owner, pkg = parse_package(package)
+    session = _get_session(team)
 
-    session.put("%s/api/access/%s/%s/%s" % (get_registry_url(), owner, pkg, user))
+    session.put("%s/api/access/%s/%s/%s" % (get_registry_url(team), owner, pkg, user))
 
 def access_remove(package, user):
     """
     Remove access
     """
-    owner, pkg = parse_package(package)
-    session = _get_session()
+    team, owner, pkg = parse_package(package)
+    session = _get_session(team)
 
-    session.delete("%s/api/access/%s/%s/%s" % (get_registry_url(), owner, pkg, user))
+    session.delete("%s/api/access/%s/%s/%s" % (get_registry_url(team), owner, pkg, user))
 
 def delete(package):
     """
     Delete a package from the server.
 
     Irreversibly deletes the package along with its history, tags, versions, etc.
     """
-    owner, pkg = parse_package(package)
+    team, owner, pkg = parse_package(package)
+    if team:
+        teamstr = "%s:" % team
+    else:
+        teamstr = ""
 
     answer = input(
         "Are you sure you want to delete this package and its entire history? " +
-        "Type '%s/%s' to confirm: " % (owner, pkg)
+        "Type '%s%s/%s' to confirm: " % (teamstr, owner, pkg)
     )
 
-    if answer != '%s/%s' % (owner, pkg):
+    if answer != '%s%s/%s' % (teamstr, owner, pkg):
```

Comment:
another example of the fmt string

Reply:
if we want to change it we can grep teamstr, no reason to refactor imo